src/modules/fancyzones/editor/FancyZonesEditor/App.xaml.cs

```diff
@@ -67,7 +74,7 @@ private void DebugModeCheck()
 
         public App()
         {
-            DebugModeCheck();
+            // DebugModeCheck();
```

Comment:
Why is it commented out? Should we remove it?

Reply:
We still want to enable debug mode for stand alone debugging, we can revisit this in 0.33 or 0.35.